streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionResponseWrapper.java

```diff
@@ -21,22 +21,30 @@
 import java.util.Arrays;
 
 public class SubscriptionResponseWrapper<FV> {
-    final static byte CURRENT_VERSION = 0x00;
+    final static byte CURRENT_VERSION = 0x01;
+    // 0x00
```

Comment:
nit:
```
// version 0x00 fields:
```

Same below:
```
// version 0x01 fields:
```

Or just change to `1` instead of `0x01` (compare Subscription.java -- might be nice to align both).

Reply:
Fixed in https://github.com/apache/kafka/pull/11945/commits/f7f9181ec3312b21a3534919819ccd02dfdcbc74#diff-4a79c25b7d01993f00f2ffa554e15f18a2c4f6e1e0f5c18dc33ad24015261f22R24-R29